### DISCUSSION TITLE
`All: unset` is a "nice-to-have" utility

### DISCUSSION THREAD

**Discussion by frykten** [tailwindlabs/tailwindcss#11594] [Ideas]
Currently, there is no `all-unset` class in Tailwind, and to be able to use the CSS property, we need to use the `[all:unset]` class, which results in a higher up priority (it is created afterwards and has the classic 10pts priority for classes, thus resetting all other 10pts classes, thus nulling the good effects because of all the bad that comes of resetting the style you are trying to add...

What do you think? It is not "game changer", but it is a nice softie, imho.

**Comment by frykten** [comment]
Meh, we just hadn't (finished) configured well our new tailwind.

Sorry for the bother.

**Comment by equivalent** [comment]
I'm transitioning established project to TW, I need to do this with preflight turned off. So many times I need to disable styles on parts of project. 

This is where unset is really helpful

**Comment by jkazimierczak** [comment]
This also comes useful when working with TailwindCSS-based component libraries, that do what you want, but have so many styles, that clearing them one by one is just tedious.

**Comment by asyncawaitpromise** [comment]
Still asking for this utility. I'd use it often.

**Comment by tallneil** [comment]
+1 we need unset!

**Comment by jfbrennan** [comment]
I use this in my homegrown design system all the time when I need an unstyled button or link:
```html
<button class="all-unset pointer">...</button>
```
Was surprised at work yesterday when I went to type `all-unset` and learned TW doesn't have it :(
